obspy/clients/httpproxy.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+"""
+Establish a socket connection through an HTTP proxy.
+Author: Fredrik Østrem <frx.apps@gmail.com>
+License:
+  This code can be used, modified and distributed freely, as long as it is this
+  note containing the original author, the source and this license, is put
+  along with the source code.
+
+J. MacCarthy, modified from https://gist.github.com/frxstrem/4487802
+
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+from future import standard_library
+from future.utils import native_str
+
+from base64 import b64encode
+import socket
+with standard_library.hooks():
+    import urllib
```

Comment:
These imports are backwards; it's using the Python 2 name, not the Python 3 name.